### PR TITLE
Decrease file size and project size limits

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: cli/cmd/deploy.go [hunk 1]
```diff
@@ -44,8 +44,8 @@ var (
 	_warningProjectBytes = 1024 * 1024 * 10
 	_warningFileCount    = 1000
 
-	_maxFileSizeBytes    int64 = 1024 * 1024 * 512
-	_maxProjectSizeBytes int64 = 1024 * 1024 * 512
+	_maxFileSizeBytes    int64 = 1024 * 1024 * 50 // 50mb
+	_maxProjectSizeBytes int64 = 1024 * 1024 * 50 // 50mb
```
Comment (RobertLucian) [inline cli/cmd/deploy.go:48]: Let's make it 32MiB.

Comment (vishalbollu) [reply cli/cmd/deploy.go:48]: Changing these limits would cause the CLI to throw an explicit error. The deploy request won't even be sent to the operator.

**Review by vishalbollu — APPROVED** [review]

File: cli/cmd/deploy.go (hunk 1 above)
Comment (deliahu) [reply cli/cmd/deploy.go:48]: @maskani-moh When you were running into the OOM issue caused by your large files, did the CLI show you a warning about the file size? If not, were you by chance using the `--yes` flag or the Python client?

Comment (maskani-moh) [reply cli/cmd/deploy.go:48]: I was indeed using the `--yes` flag, so the CLI wasn't complaining about the large files.

**Pull request merged**